### PR TITLE
Make reaction buttons persistent

### PR DESCRIPTION
**Pull request by AlphaKretin**
## Description

Loads reaction buttons based on saved Register Messages on bot launch. This feature was motivated by wanting to restart just before reacting to the message in order to not be in the member cache while testing changes to that system, however, in general, more persistence should be better. Unfortunately, this is currently not working because we receive a 401 Unauthorised error upon trying to get the discord message, which I don't entirely understand because a similar process for Persistent Timers has worked fine.

## Checklist

- [x] I am following the [contributing guidelines]( https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: src/database/postgres.ts [hunk 1]
```diff
@@ -269,11 +269,18 @@ export class DatabaseWrapperPostgres implements DatabaseWrapper {
...
+				]
+			});
+			return tournaments.map(t => this.wrap(t));
+		}
 		const tournaments = await ChallongeTournament.find({
-			where: [
-				{ owningDiscordServer, status: TournamentStatus.IPR },
-				{ owningDiscordServer, status: TournamentStatus.PREPARING }
-			]
+			where: [{ status: TournamentStatus.IPR }, { status: TournamentStatus.PREPARING }]
```
Comment (kevinlul) [inline src/database/postgres.ts:283]: Does the query return an empty array with an undefined parameter?

File: src/timer.ts [hunk 2]
```diff
@@ -117,10 +117,10 @@ export class PersistentTimer {
...
 			const left = PersistentTimer.formatTime(this.entity.end.getTime() - Date.now());
-			try {
-				const message = await this.discord.getMessage(this.entity.channelId, this.entity.messageId);
+			const message = await this.discord.getMessage(this.entity.channelId, this.entity.messageId);
+			if (message) {
 				message.edit(`Time left in the round: \`${left}\``);
-			} catch (err) {
+			} else {
 				logger.warn(`${this.entity.channelId} ${this.entity.messageId} was removed`);
 			}
```
Comment (kevinlul) [inline src/timer.ts:125]: Does this introduce the potential for throwing an exception here that we don't want?

File: src/database/postgres.ts (hunk 1 above)
Comment (AlphaKretin) [reply src/database/postgres.ts:283]: Yes, that was my experience.

File: src/timer.ts (hunk 2 above)
Comment (AlphaKretin) [reply src/timer.ts:125]: It likely does, we should probably still have error handling here, but assuming any error is "the message doesn't exist anymore" will just lead to absorption.

**Review by kevinlul — APPROVED** [review]
lol approving own code

**Comment by AlphaKretin** [comment]
Apparently, I can't approve this since it's my own PR, but for the record, I do approve @kevinlul's changes.

**Pull request merged**